### ISSUE TITLE
Add Menu Help to User Documentation (Trac #512)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#645]
For new users, help on understanding what is under each menu item would be useful.  A new top level entry in User Documentation needs to be created and documentation written for it. 

Migrated from http://trac.sasview.org/ticket/512
```json
{
    "status": "closed",
    "changetime": "2016-10-07T00:14:22",
    "_ts": "2016-10-07 00:14:22.288508+00:00",
    "description": "For new users, help on understanding what is under each menu item would be useful.  A new top level entry in User Documentation needs to be created and documentation written for it. ",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2016-03-15T17:03:50",
    "component": "SasView",
    "summary": "Add Menu Help to User Documentation",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "smk78",
    "type": "enhancement"
}
```

**Comment by smk78** [comment]
Trac update at `2016/10/05 02:58:01`: 
* **smk78** changed owner from "" to "smk78"
* **smk78** changed status from "new" to "assigned"

**Comment by smk78** [comment]
Trac update at `2016/10/06 02:31:38`: 
* **smk78** commented:

In changeset ec860a8f303938f6a9f5b6109d0b0ec4eb7ab109:

```text
#!CommitTicketReference repository="sasview" revision="ec860a8f303938f6a9f5b6109d0b0ec4eb7ab109"
Created documentation on Menu Bar with links to Analysis, Tools and the
Tutorial. Fixes #645
```


* **smk78** changed resolution from "" to "fixed"
* **smk78** changed status from "assigned" to "closed"

**Comment by butlerpd** [comment]
Trac update at `2016/10/07 00:14:22`: **butler** changed milestone from "SasView Next Release +1" to "SasView 4.1.0"
